public/docs/v2/components/paths/seal/update.yaml

```
@@ -0,0 +1,25 @@
+tags:
+  - Selos
+summary: Atualiza um selo
+description: Atualiza um selo no sistema com as informações fornecidas.
+parameters:
+  - in: path
+    name: id
+    required: true
+    description: ID do selo a ser atualizado
+    example: 1
```

Você pode substituir esse trecho por : 
```suggestion
  - $ref: '../../parameters/path/id.yaml'
```